tests/prepare/feature/basic/test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+. ../../../images.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "PROVISION_HOW=${PROVISION_HOW:-container}"
+
+        build_container_image "fedora/latest/upstream\:latest"
+
+        rlRun "run=\$(mktemp -d)" 0 "Create run directory"
+        rlRun "pushd data"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Test whether unserialization works"
+        rlRun "tmt -vv run --id $run --scratch --until prepare"
```

Comment:
I fail to reproduce the bug without including `prepare --how feature --profile PROFILE` on this line. So I don't think this test exercises the area that's being fixed in this patch.